vlib/x/encoding/asn1/DOCS.md

```diff
@@ -0,0 +1,496 @@
+# `asn1` module documentation.
+
+> [!CAUTION]
+> This module is marked as an experimental, so, its a subject to change (even rapidly).
+> Use it with caution, submit when found a bug and gives yours feedback and review.
```

Comment:
That section should go into `README.md` .
The reason is that `README.md` files in modules, are included to the documentation rendered at https://modules.vlang.io/ .

Reply:
I would add it into readme file

Reply:
Done